### PR TITLE
convert the BccPin component to use css classes

### PR DESCRIPTION
**Pull request by jasverix**
+ removed xxs size since it does not exist in tailwind

### CODE REVIEW

**Comment by u12206050** [comment]
Xxs is an additional size we added.

**Comment by jasverix** [comment]
The class text-xss does not exist, so it crashed while trying to move it to CSS with @apply.

**Comment by jasverix** [comment]
Suggestion from @laurensgroeneveld is to just remove the size prop for now - it seems to not be used in this component

**Comment by u12206050** [comment]
@laurensgroeneveld xxs needs to be added then with size 10px according to the design system.

**Comment by laurensgroeneveld** [comment]
Right, actually all the typography needs to be added, it's a thing on our todo. We've decided to remove all the sizes for now since the design only specifies one size, and it doesn't seem the `BccPin` is used anywhere (@u12206050 maybe you can confirm that? Can't find it in the event app).

File: src/css/pin.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+@layer components {
+    .bcc-pin {
+        @apply inline-block rounded-full p-[.3em] shadow-md text-xs;
+    }
+
+    .bcc-pin .bcc-pin-icon {
+        @apply h-[1.2em] w-[1.2em]
```
Comment (laurensgroeneveld) [inline src/css/pin.css:7]: Since we only have the one size from the design now, this could/should be `h-4 w-4`.

**Pull request merged**